src/App.vue

```diff
@@ -21,6 +22,14 @@ export default defineComponent({
     const store = useStore();
     store.dispatch("INIT_VUEX");
 
+    const route = useRoute();
+    const query = computed(() => route.query);
+    watch(query, (newQuery) => {
+      if (newQuery) {
+        store.dispatch("SET_IS_SAFE_MODE", newQuery["isSafeMode"] === "true");
+      }
+    });
```

Comment:
一瞬何やっているかわかりにくいかもなので、コメント書いときますか。
```suggestion
    // セーフモードにする
    const route = useRoute();
    const query = computed(() => route.query);
    watch(query, (newQuery) => {
      if (newQuery) {
        store.dispatch("SET_IS_SAFE_MODE", newQuery["isSafeMode"] === "true");
      }
    });
```